src/rgw/rgw_sal_motr.cc

```diff
@@ -166,7 +166,7 @@ int MotrMetaCache::remove(const DoutPrefixProvider *dpp,
   ObjectCacheInfo info;
   int rc = distribute_cache(dpp, name, info, INVALIDATE_OBJ);
   if (rc < 0) {
-    ldpp_dout(dpp, 0) << "ERROR: " <<__func__<< "(): failed to distribute cache: rc =" << rc << dendl;
+    ldpp_dout(dpp, 0) << "ERROR: " <<__func__<< ": failed to distribute cache: rc = " << rc << dendl;
```

Comment:
```suggestion
    ldpp_dout(dpp, 0) << "ERROR: " << __func__ << ": failed to distribute cache: rc = " << rc << dendl;
```

Reply:
For now I going ahead. this change can be taken later.